setup.py

```diff
@@ -26,6 +27,10 @@
 with open(os.path.join(here, "README.md"), "r", "utf-8") as f:
     readme = f.read()
 
+orjson_require = []
+if platform.python_implementation() != 'PyPy':
+    orjson_require = ['orjson >= 3.11.0']
+
```

Comment:
```suggestion

```

Instead of having logic here and adding it to `install_requires` later, you can use [Environment Marker](https://packaging.python.org/en/latest/specifications/dependency-specifiers/#environment-markers)
```python
install_requires=[
    ...
    "zstandard",
    "orjson >= 3.11.0 ; platform_python_implementation != 'PyPy'"
],
 ```